constantTimeCompare.js

```diff
@@ -30,18 +30,17 @@ module.exports = function constantTimeCompare(a, b) {
   // Using with{} nixes some V8 optimizations that would otherwise undermine
   // our intentions here.
   with({}) {
-    // Add at least one character so that there's at least one thing to modulo over.
-    a += " ";
-    b += " ";
     var aLen = a.length,
         bLen = b.length,
-        match = aLen === bLen ? 1 : 0,
+        match = 0,
         i = Math.max(aLen, bLen, MAX_KEY_CHARS);
+    // If the lengths don't match, the string doesn't match
```

Comment:
This comment just repeats what the next line of code says, so I think it can safely be removed.